src/D2L.CodeStyle.Analyzers/Immutability/MutabilityInspector.cs

```diff
@@ -517,19 +523,34 @@ private static T GetDeclarationSyntax<T>( ISymbol symbol )
 			return decl;
 		}
 
-		private MutabilityInspectionResult InspectMemberRecursive(
+		private MutabilityInspectionResult InspectMember(
 			ISymbol symbol,
 			HashSet<ITypeSymbol> typeStack
 		) {
-			// if the member is audited or unaudited, ignore it
+			// if the member is audited or unaudited, check if valid
 			if( Attributes.Mutability.Audited.IsDefined( symbol ) ) {

```

Comment:
Is the plan for there to be a second PR for unaudited? That's fine. I wouldn't mind it in this PR though because it'd keep code that should be mostly symmetric, mostly symmetric.

Reply:
Sounds good I'll add unauditeds to this PR as well. Planning to emit the diagnostics separately though

Reply:
Yep that's good